pyproject.toml

```
@@ -13,3 +13,15 @@ djlint = "1.32.0"
 [tool.djlint]
 profile="golang"
 ignore="H005,H006,H008,H013,H016,H020,H021,H030,H031"
+
+[tool.codespell]
+skip = '.git,*.pdf,*.svg,package-lock.json,go.mod,locale,license,*.git,objects,*.fr-fr.*,*.de-de.*,*.css,go.sum,*.key,gitignore,pyproject.toml,diff_test.go,go-licenses.json'
+# precise hits for CamelCased words,various other curious cases which require regex to ignore
+# entire line or some portion of it
+ignore-regex = '(\b(mx claus|commitT|ReadBy|#afile|respOne|commitI|[cC]rossReference)\b|shouldbe\.|women’s.*womens|"emoji":.*|,bu,|assert\.Equal.*"fo\b|github\.com/unknwon|Copyright 2014 Unknwon|allowed\.noone|[hH]eadErr|atLeast|{"\\U.*|\.metdata)'
```

if https://github.com/go-gitea/gitea/pull/26207 gets merged before this
```suggestion
ignore-regex = '(\b(mx claus|commitT|ReadBy|#afile|respOne|commitI|[cC]rossReference)\b|shouldbe\.|women’s.*womens|"emoji":.*|,bu,|assert\.Equal.*"fo\b|github\.com/unknwon|Copyright 2014 Unknwon|allowed\.noone|[hH]eadErr|atLeast|{"\\U.*)'
```